.eslintrc.js

```diff
@@ -63,53 +50,33 @@ module.exports = {
     'no-empty': 'off',
     'arrow-parens': 'off',
     'padded-blocks': 'off',
-
     'jest/expect-expect': 'off',
-
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'warn',
-
     'react/display-name': 'off',
     'react/prop-types': 'off',
-
     'prettier/prettier': 'error',
-
     'space-before-function-paren': 'off',
     'generator-star-spacing': 'off',
-
-    'import/order': [
-      'error',
-      {
-        alphabetize: {
-          order: 'asc'
-        },
-        groups: [
-          'builtin',
-          'external',
-          'internal',
-          'parent',
-          'sibling',
-          'index'
-        ],
-        'newlines-between': 'always',
-        pathGroups: [
-          {
-            pattern: 'react',
-            group: 'builtin',
-            position: 'before'
-          }
-        ],
-        pathGroupsExcludedImportTypes: ['builtin']
-      }
-    ],
+    'import/order': ['error', {
+      alphabetize: {
+        order: 'asc'
+      },
+      groups: ['builtin', 'external', 'internal', 'parent', 'sibling', 'index'],
+      'newlines-between': 'always',
+      pathGroups: [{
+        pattern: 'react',
+        group: 'builtin',
+        position: 'before'
+      }],
+      pathGroupsExcludedImportTypes: ['builtin']
+    }],
     'import/no-default-export': 'error'
   },
-  overrides: [
-    {
-      files: ['src/**/*.d.ts', 'src/**/*.stories.tsx'],
-      rules: {
-        'import/no-default-export': 'off'
-      }
+  overrides: [{
+    files: ['src/**/*.d.ts', 'src/**/*.stories.tsx'],
+    rules: {
+      'import/no-default-export': 'off'
     }
-  ]
-}
+  }]
+};
```

Comment:
Did prettier run on this? Weird that it added a semicolon and dropped the trailing space

Reply:
ohhh hm that is weird. ill take another look